win32ss/user/user32/windows/mdi.c

```diff
@@ -2147,7 +2147,13 @@ CascadeWindows(HWND hwndParent, UINT wFlags, LPCRECT lpRect,
             continue;
 
         if (IsZoomed(hwnd))
-            ShowWindow(hwnd, SW_RESTORE | SW_SHOWNA);
+        {
```

Comment:
Technically the show window should be enough; if the window to show doesn't respond, it should just be ghosted by win32k. So it seems that here this is a workaround for an actual still unfixed bug.

Reply:
Reverted.